apollo-compiler/src/test/kotlin/com/apollographql/apollo/compiler/CodegenTest.kt

```diff
@@ -73,6 +73,10 @@ class CodeGenTest(val folder: File) {
       } else {
         emptyMap()
       }
+      val singularizeTypes = when (folder.name) {
```

Comment:
Wouldn't this be more simple if you just do `folder.name == "singularize_types"`

Reply:
Yes, but every other check here uses when, so I made sure my code fit in with the existing patterns.

Reply:
I would personally prefer simpler solutions over consistency just for consistency's sake. 😅 Resolving this now since it is personal opinions at the end.